### PR TITLE
Create springer-imis-series-migrationsgesellschaften-english.csl

### PR DESCRIPTION
**Pull request by Schrader311**
This stylesheet is for the ongoing book series 'Migrationsgesellschaften' at the Institute for Migration Research and Intercultural Studies at University Osnabrück, Germany. It is especially for the English-writing authors and should help to institutionalize the ongoing book series. The style fits the requirements of the publisher.

### CODE REVIEW

**Comment by adam3smith** [comment]
Still trying to understand the relationship between this and the various closed pull request. From my perspective, the ideal scenario would still seem to be a single, language neutral citation style.

**Comment by Schrader311** [comment]
I'd love to code a language-neutral citation style, but unfortunately, due to my coding skills (which are not existing), I simply was not able to code one of this kind. To me, it feels like a big problem of Zotero/CSL that there are no additional manuals, which could help me to build the required citation style. As this is a project I am working on in my working time, I am not able to teach myself the whole process of coding a citation style, nor will I be able to put any more working time into it. Sadly none of my coworkers has these skills, so we have to deal with the outcomes I was able to produce. We, therefore, decided to reduce ourselves to the citation style in English, as we need this one by the end of the month Septembre. We hope that you understand and support our request.

**Comment by Schrader311** [comment]
@adam3smith may you please reply to my statement or better, may you just accept the stylesheet? We really need it for our further work on the book series.

Thank you very much.

File: springer-imis-series-migrationsgesellschaften.csl [hunk 1]
```diff
@@ -0,0 +1,466 @@
...
+          <group delimiter=" ">
+          <text term="accessed" text-case="capitalize-first"/>
+                        <date variable="accessed" form="text" date-parts="year-month-day"/>
+          </group>
+        </else-if>
+      </choose>
+    </group>
+    <choose>
+      <if type="webpage" match="any">
+        <text variable="abstract"/>
```
Comment (adam3smith) [inline springer-imis-series-migrationsgesellschaften.csl]: This is a bit puzzling -- what do you want the abstract for for webpages?

Comment (Schrader311) [reply springer-imis-series-migrationsgesellschaften.csl]: @adam3smith thank you so much for your help! To be honest, we do not need abstracts for web pages, I can not explain why that's in the code. But the line can be deleted. 
Once again, many thanks for your support

**Comment by adam3smith** [comment]
OK, removed, thanks. Sorry for the long wait.

**Comment by rmzelle** [comment]
@Schrader311, we forgot to ask: are the publisher guidelines available online somewhere? We try to always include "documentation" links in every style (see also https://github.com/citation-style-language/styles/wiki/Style-Requirements#10---documentation-link).

**Comment by Schrader311** [comment]
@rmzelle, this is the link for the book series: https://www.imis.uni-osnabrueck.de/publikationen/migrationsgesellschaften.html There are no publishers guidelines online yet, but we will put it online in the next weeks.

**Comment by Schrader311** [comment]
@adam3smith @rmzelle I would like to thank you both very much for your support in completing this task. Without your efforts, it would not have been finished! Thanks

**Pull request merged**